samples/todo-app/stats-api/server.js

```diff
@@ -13,6 +13,11 @@ var cache = redis.createClient({
     password: process.env.REDIS_PASSWORD || undefined
 });
 
+app.get('/hello', function (req, res) {
```

Comment:
Nit: should we use '/stats/hello', or something like that, as this is the probe for stats-api, and not for the whole app? That said, it's a sample app so it's not a big deal. Feel free to won't fix.

Reply:
This endpoint is on the stats-api only. It is not available on any other microservice.